File: Codeforces/C++/problemset/Holiday Of Equality.cpp
// Problem Link: https://codeforces.com/problemset/problem/758/A

#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
    ll n, sum = 0, max = 0;
    cin >> n;
    ll *x = new ll[n];
    for(ll i = 0; i < n; i++)
    {
        cin >> x[i];
        if(x[i] > max) max = x[i];
    }
    for(ll i = 0; i < n; i++)
        sum += (max-x[i]);
    cout << sum;
    delete []x;
    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/HolidayOfEquality.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/758/A */

fun main() {
    val n = readln()
    val welfare = readln().split(" ").map { it.toInt() }
    val max = welfare.max()
    var sum = 0

    welfare.forEach {
        sum += max - it
    }

    print(sum)
}